lua/lean/rpc.lua

```diff
@@ -86,7 +87,8 @@ local function connect(bufnr)
 end
 vim.lsp.handlers['$/lean/rpc/connected'] = function(err, _, params, _, _, _)
   if err ~= nil then return end
-  local bufnr = vim.uri_to_bufnr(params.uri)
```

Comment:
This is not a big deal, it's just that `uri_to_bufnr` looks like it can create a new buffer if one doesn't already exist for that URI, which might be unexpected.